oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -396,10 +396,13 @@ public APIGatewayProxyResponseEvent handleRequest(
                                     ? OrchSessionItem.AccountState.NEW
                                     : OrchSessionItem.AccountState.EXISTING;
                 }
-                sessionService.storeOrUpdateSession(userSession.setNewAccount(accountState));
-                orchSessionService.updateSession(orchSession.withAccountState(orchAccountState));
+                userSession = userSession.setNewAccount(accountState);
+                orchSession = orchSession.withAccountState(orchAccountState);
 
-                sessionService.storeOrUpdateSession(userSession.setAuthenticated(true));
+                userSession = userSession.setAuthenticated(true);
```

Comment:
The last commit is making a change to lines changed in the second commit. I think these lines refactored in the final commit can just be refactored in the second commit instead

Reply:
Fixed :)